packages/deepbook/sources/pool.move

```diff
@@ -200,6 +206,9 @@ module deepbook::pool {
             (base_quantity, _, _) = self.get_quantity_out(0, quote_quantity, clock);
         };
         base_quantity = base_quantity - base_quantity % self.load_inner().book.lot_size();
+        if (base_quantity < self.load_inner().book.min_size()) {
+            return (base_in, quote_in, deep_in)
```

Comment:
Should we error out here instead of returning? If there's no error there might be an assumption that the swap happened